test/resources/weather_data_train.csv

```diff
@@ -0,0 +1 @@
+How hot is it today?,temperature
Is it hot outside?,temperature
Will it be uncomfortably hot?,temperature
Will it be sweltering?,temperature
How cold is it today?,temperature
Is it cold outside?,temperature
Will it be uncomfortably cold?,temperature
Will it be frigid?,temperature
What is the expected high for today?,temperature
What is the expected temperature?,temperature
Will high temperatures be dangerous?,temperature
Is it dangerously cold?,temperature
When will the heat subside?,temperature
Is it hot?,temperature
Is it cold?,temperature
How cold is it now?,temperature
Will we have a cold day today?,temperature
When will the cold subside?,temperature
What highs are we expecting?,temperature
What lows are we expecting?,temperature
Is it warm?,temperature
Is it chilly?,temperature
What's the current temp in Celsius?,temperature
What is the temperature in Fahrenheit?,temperature
Is it windy?,conditions
Will it rain today?,conditions
What are the chances for rain?,conditions
Will we get snow?,conditions
Are we expecting sunny conditions?,conditions
Is it overcast?,conditions
Will it be cloudy?,conditions
How much rain will fall today?,conditions
How much snow are we expecting?,conditions
Is it windy outside?,conditions
How much snow do we expect?,conditions
Is the forecast calling for snow today?,conditions
Will we see some sun?,conditions
When will the rain subside?,conditions
Is it cloudy?,conditions
Is it sunny now?,conditions
Will it rain?,conditions
Will we have much snow?,conditions
Are the winds dangerous?,conditions
What is the expected snowfall today?,conditions
Will it be dry?,conditions
Will it be breezy?,conditions
Will it be humid?,conditions
What is today's expected humidity?,conditions
Will the blizzard hit us?,conditions
Is it drizzling?,conditions
```

Comment:
do we need this file?

Reply:
I included all of the resources that were used in the tests. If that one is used for JSON training data, it will get removed but not until the next major version